recipes/tttrlib/build.sh

```
@@ -0,0 +1,26 @@
+  CXXFLAGS="${CXXFLAGS} -D_LIBCPP_DISABLE_AVAILABILITY"
+fi
+
+cmake \
+ -DCMAKE_INSTALL_PREFIX="$PREFIX" \
+ -DCMAKE_PREFIX_PATH="$PREFIX" \
+ -DBUILD_PYTHON_INTERFACE=ON \
+ -DCMAKE_BUILD_TYPE=Release \
+ -DCMAKE_LIBRARY_OUTPUT_DIRECTORY="$SP_DIR" \
+ -DCMAKE_SWIG_OUTDIR="$SP_DIR" \
+ -DBUILD_LIBRARY=ON \
+ -DPYTHON_VERSION=$(python -c 'import platform; print(platform.python_version())')\
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)